csrc/host_ir/evaluator.cpp

```diff
@@ -284,12 +285,9 @@ void HostIrEvaluator::handle(PostOnStream* post_ir) {
                hu->fusion_to_execute(), 1, 1, 1, 1, SchedulerType::None)});
       ExecutorAbstract* ea = it2.first->second.get();
       if (ea->isA<KernelExecutor>()) {
+        HeuristicParams params(SchedulerType::None);
```

Comment:
nitpick: I feel like the API change is unobvious to the HostIR folks.
```suggestion
        HeuristicParams default_params(SchedulerType::None);
```